<launch>
    <arg name="face1"       default="child_purple" />
    <arg name="face2"       default="child_yellow" />

    <param name="$(arg face1)_description" command="$(find xacro)/xacro --inorder $(find freeplay_sandbox)/urdf/face.xacro name:=$(arg face1)" />
    <param name="$(arg face2)_description" command="$(find xacro)/xacro --inorder $(find freeplay_sandbox)/urdf/face.xacro name:=$(arg face2)" />

    <arg name="camera1"     default="camera_purple" />
    <arg name="camera2"     default="camera_yellow" />

    <group ns="$(arg camera1)">
        <node pkg="gazr" type="estimate" name="gazr" required="true">
            <param name="face_model" value="$(find gazr)/shape_predictor_68_face_landmarks.dat" />
            <param name="prefix" value="$(arg face1)" />
            <remap from="image" to="rgb/image_raw"/>
        </node>

        <!--        <node name="estimate_focus" pkg="freeplay_sandbox" type="estimate_focus" output="screen"/> -->
    </group>

    <group ns="$(arg camera2)">
        <node pkg="gazr" type="estimate" name="gazr" required="true">
            <param name="face_model" value="$(find gazr)/shape_predictor_68_face_landmarks.dat" />
            <param name="prefix" value="$(arg face2)" />
            <remap from="image" to="rgb/image_raw"/>
        </node>

        <!--<node name="estimate_focus" pkg="freeplay_sandbox" type="estimate_focus" output="screen"/>-->
    </group>

</launch>
